### DISCUSSION TITLE
Supporter Certificate when it doesn't prevent detection of VM or hypervisor?

### DISCUSSION THREAD

**Discussion by acidhorse1972** [sandboxie-plus/Sandboxie#3076] [Various ideas and suggestions]
Thinking about getting a Supporter Certificate when it finally offers prevention of detection of VM or hypervisor.
That's really what I'm waiting on.

**Comment by isaak654** [comment]
@DavidXanatos recently shared a thought about this:
> The use case is software which dont want to run under sandboxie or in a VM in general and the user is looking for a way to use sandboxie to run such an application in a vm. The required effort to make this work is huge

See issue #234 for further replies.
